libs/scripts/scripts.js

```diff
@@ -18,17 +18,20 @@ import {
   loadDelayed,
   loadTemplate,
   setConfig,
+  getLocale,
 } from '../utils/utils.js';
 
+const locales = {
+  '': { ietf: 'en-US', tk: 'hah7vzn.css' },
+  de: { ietf: 'de-DE', tk: 'hah7vzn.css' },
+  cn: { ietf: 'zh-CN', tk: 'tav4wnu' },
+  kr: { ietf: 'ko-KR', tk: 'zfo3ouc' },
+};
 const config = {
   imsClientId: 'milo',
-  projectRoot: `${window.location.origin}/libs`,
-  locales: {
-    '': { ietf: 'en-US', tk: 'hah7vzn.css' },
-    de: { ietf: 'de-DE', tk: 'hah7vzn.css' },
-    cn: { ietf: 'zh-CN', tk: 'tav4wnu' },
-    kr: { ietf: 'ko-KR', tk: 'zfo3ouc' },
-  },
+  scriptsRoot: `${window.location.origin}/libs`,
+  contentRoot: `${window.location.origin}${getLocale(locales).prefix}`,
```

Comment:
I think we can clean this up.

There are outliers for both contentRoot & scriptsRoot, but most consumers won't need either. I think Milo really only needs scriptsRoot here so we know where to loadBlocks from.

Most consumers are going to be:

```js
// blog.adobe.com
contentRoot = window.location.origin
scriptsRoot = window.location.origin
```
Now, we have other projects, where things deviate:

```js
// business.adobe.com/blog
contentRoot = window.location.origin/blog
scriptsRoot = window.location.origin
```
This should actually change with the BACOM project...

```js
// business.adobe.com/blog
contentRoot = window.location.origin/blog
scriptsRoot = window.location.origin/blog
```

Because BACOM should take over:

```js
// business.adobe.com
contentRoot = window.location.origin
scriptsRoot = window.location.origin // /scripts/scripts.js, /blocks/my-block/my-block.css
```
In the case of stock.adobe.com, I think both need to change:
```js
// business.adobe.com
contentRoot = window.location.origin/pages
scriptsRoot = window.location.origin/pages // /scripts/scripts.js, /blocks/my-block/my-block.css
```
**I think we're using this in three scenarios:**
1. We need FQDN's for loading styles & scripts.
2. We need to know certain things for gnav search.
3. Times when we need to pull default content (/seo/placeholders.json)

All of these thoughts makes me think we can get away with...

Milo
scriptsRoot: /libs

Blog
{intentionally left blank}

BACOM
{intentionally left blank}

BACOM Blog
scriptsRoot: /blog
contentRoot: /blog

Adobe Stock / Artist Hub
scriptsRoot: /pages
contentRoot: /pages